maa-cli/src/config/task/client_type.rs

```diff
@@ -33,6 +35,37 @@ impl ClientType {
         }
     }
 
+    pub fn reset_time(&self) -> DateTime<Local> {
```

Comment:
I believe this function could return `chrono::NaiveTime::from_hms_opt(4, 0, 0).unwrap()`, and then be compared with `datetime.time()` in the `game_date` function.